tests/Paramore.Brighter.RMQ.Async.Tests/MessagingGateway/Proactor/When_posting_a_message_via_the_messaging_gateway_quorum_async.cs

```diff
@@ -0,0 +1,141 @@
+using Paramore.Brighter.MessagingGateway.RMQ.Async;
+using Paramore.Brighter.Observability;
+using Xunit;
+
+namespace Paramore.Brighter.RMQ.Async.Tests.MessagingGateway.Proactor;
+
+[Trait("Category", "RMQ")]
+public class RmqMessageProducerSendMessageQuorumTestsAsync : IDisposable, IAsyncDisposable
+{
+    private readonly IAmAMessageProducerAsync _messageProducer;
+    private readonly IAmAMessageConsumerAsync _messageConsumer;
+    private readonly Message _message;
+
+    public RmqMessageProducerSendMessageQuorumTestsAsync()
+    {
+        var messageId = Guid.NewGuid().ToString();
+        var topic = new RoutingKey(Guid.NewGuid().ToString());
+        var messageType = MessageType.MT_COMMAND;
+        var source = new Uri("http://testing.example");
+        var type = "test-type";
+        var timestamp = DateTimeOffset.UtcNow;
+        var correlationId = Guid.NewGuid().ToString();
+        var replyTo = new RoutingKey("reply-queue");
+        var contentType = new ContentType(MediaTypeNames.Text.Plain);
+        var handledCount = 5;
+        var dataSchema = new Uri("http://schema.example");
+        var subject = "test-subject";
+        var delayed = TimeSpan.FromSeconds(30);
+        var traceParent = "00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01";
+        var traceState = "congo=t61rcWkgMzE";
+        var baggage = new Baggage();
+        baggage.LoadBaggage("userId=alice");
+
+        _message = new Message(
+            new MessageHeader(
+                messageId: messageId,
+                topic: topic,
+                messageType: messageType,
+                source: source,
+                type: type,
+                timeStamp: timestamp,
+                correlationId: correlationId,
+                replyTo: replyTo,
+                contentType: contentType,
+                handledCount: handledCount,
+                dataSchema: dataSchema,
+                subject: subject,
+                delayed: delayed,
+                traceParent: traceParent,
+                traceState: traceState,
+                baggage: baggage),
+            new MessageBody("test content"));
+
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
+        };
+
+        _messageProducer = new RmqMessageProducer(rmqConnection);
+        var queueName = new ChannelName(Guid.NewGuid().ToString());
+            
+        // Create consumer with Quorum queue type, ensuring durability is enabled
+        // The consumer will handle queue creation automatically
+        _messageConsumer = new RmqMessageConsumer(rmqConnection, queueName, _message.Header.Topic, 
+            isDurable: true, // Required for quorum queues
+            highAvailability: false, // Not supported for quorum queues
+            queueType: QueueType.Quorum);
+    }
```

Comment:
You need a call to QueueFactory, as follows:

` new QueueFactory(rmqConnection, queueName, RoutingKeys(_message.Header.Topic))
            .CreateAsync()
            .GetAwaiter()
            .GetResult();
`